doc/platform/app/app_roles.rst

```diff
@@ -182,6 +181,41 @@ In the example below, the ``apply()`` function uses the :ref:`log <log-module>`
 
 
 
+.. _roles_create_custom_role_edit:
+
+Editing a role
+~~~~~~~~~~~~~~
+
+Editing a custom application role object means changing the contents of its LUA file. To edit a role object, you can do the following:
+
+* Edit, remove, or set to nil the the role configuration schema;
+
+* Edit, remove, or set to nil dependencies;
+
+* Edit, remove, or set to nil the ``on_event`` callback function;
+
+* Edit the role configuration validation function;
+
+* Edit the apply validation function;
+
+* Edit the stopping role function.
+
+..  code-block:: lua
+
+    return {
+        validate = function() -- ... -- end,
+        apply = function() -- ... -- end,
+        stop = function() -- ... -- end,
+        dependencies = { -- ... -- },
+        on_event = nil
+        end,
+    }
+
+
+Once the editing of the role object is finished, the :ref:`config:reload() <config_api_reference_reload>` is triggered automatically
```

Comment:
It is not exactly so. the reloading from a remote configuration storage is triggered automatically if the remote storage is used. But no, `config:reload()` is not called. These are a bit different things. At least, `config:reload()` re-reads all the configuration sources, including a file, while the automatic reload doesn't do it.

Reply:
Irrelevant after the latest update.